sentinel-adapter/sentinel-web-servlet/src/main/java/com/alibaba/csp/sentinel/adapter/servlet/CommonFilter.java

```diff
@@ -42,18 +42,27 @@
 /***
  * Servlet filter that integrates with Sentinel.
  *
+ * @author zhaoyuguang
  * @author youji.zj
  * @author Eric Zhao
  */
 public class CommonFilter implements Filter {
 
     private final static String HTTP_METHOD_SPECIFY = "HTTP_METHOD_SPECIFY";
+    /**
+     * Use the path of the url as the context, if necessary, but pay attention to the number of context EntranceNode
+     */
+    public final static String WEB_CONTEXT_UNIFY = "WEB_CONTEXT_UNIFY";
     private final static String COLON = ":";
     private boolean httpMethodSpecify = false;
+    private boolean webContextUnify = true;
 
     @Override
     public void init(FilterConfig filterConfig) {
         httpMethodSpecify = Boolean.parseBoolean(filterConfig.getInitParameter(HTTP_METHOD_SPECIFY));
+        if (filterConfig.getInitParameter(WEB_CONTEXT_UNIFY) != null) {
+            webContextUnify = Boolean.parseBoolean(filterConfig.getInitParameter(WEB_CONTEXT_UNIFY));
```

Comment:
What if an error occurs when `Boolean.parseBoolean(xxx)`? Maybe we need to handle the exception here?

Reply:
I thought so before, but I later found out that
The data type stored by filterConfig initParameter is <String, String>,
parseBoolean's method is like this
```
     Public static boolean parseBoolean(String s) {
         Return ((s != null) && s.equalsIgnoreCase("true"));
     }
```
After adding `filterConfig.getInitParameter(WEB_CONTEXT_UNIFY) != null`, I found it difficult to cause the parseBoolean method to be an error.

Reply:
👍